core/models.py

```diff
@@ -165,3 +168,30 @@ def image_url(self: "Image") -> str:
             return self.image.url
         except ValueError:
             return ""
+
+
+class CommunityMember(models.Model):

```

Comment:
Should it not be used in Community as users M2M with through model?

Reply:
This will help: https://akademiait.com.pl/zrozumiec-django-relacje-pomiedzy-modelami-wiele-do-wielu-manytomany-porady-i-triki/

Reply:
Should I drop the related manager or using both?

Reply:
ok, nvm